cumulusci/robotframework/CumulusCI.py

```diff
@@ -32,23 +32,38 @@ def __init__(self, org_name=None):
         if not org_name:
             org_name = 'dev'
         self.org_name = org_name
-        self.sf = self._init_api()
-        self.tooling = self._init_api('tooling/')
         # Turn off info logging of all http requests 
         logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.WARN)
 
     @property
-    def config(self):
-        if not hasattr(self, '_config'):
-            self._config = self._init_config()
-        return self._config
+    def project_config(self):
+        if not hasattr(self, '_project_config'):
+            self._project_config = CliConfig().project_config
```

Comment:
can you confirm that `CliConfig` will instantiate from ENV vars if present?

Reply:
I think the only ENV var that's relevant there is CUMULUSCI_KEYCHAIN_CLASS which would allow you to swap out the keychain implementation to use EnvironmentProjectKeychain.  The EnvironmentProjectKeychain is then responsible for handling the rest of the env vars.